appengine/flexible_python37_and_earlier/README.md

```diff
@@ -0,0 +1,73 @@
+## Google App Engine Flexible Environment Python Samples
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=appengine/flexible/README.md
+
+These are samples for using Python on Google App Engine Flexible Environment. These samples are typically referenced from the [docs](https://cloud.google.com/appengine/docs).
+
+See our other [Google Cloud Platform github repos](https://github.com/GoogleCloudPlatform) for sample applications and
```

Comment:
nit: any specific repos to take a look?

Reply:
@jinglundong I'd suggest just removing this line so it doesn't become a place of technical debt to update regularly.

Reply:
I don't think we want to point to any specific repos. This is generic enough to be truth for all the code samples. I'd prefer keeping them as is.

Reply:
We can work on this in a followup, but I'd suggest either

* cut it off early:
```
See our other [Google Cloud Platform github repos](https://github.com/GoogleCloudPlatform) for sample applications
```
or

* give few examples if you're mentioning specific usage like `scaffolding for other frameworks and use cases` since not all samples under `GoogleCloudPlatform` will contain such samples
```
See our other [Google Cloud Platform github repos](https://github.com/GoogleCloudPlatform) for sample applications and scaffolding for other frameworks and use cases, such as {keywords}
```